### PR TITLE
Tiding up tutorial.md

### PR DESCRIPTION
**Pull request by kpoterlowicz**

### CODE REVIEW

**Comment by kpoterlowicz** [comment]
we also like  to upload the following files to  ./../images/ 
UCSC_genome_browser
OutputHAIB450K_to_Galaxy

File: topics/epigenetics/tutorials/ewas-suite/tutorial.md [hunk 1]
```diff
@@ -26,9 +26,8 @@ The field of cancer genomics has demonstrated the power of massively parallel se
 
 ![epimechanism](../../images/Epigenetic_mechanisms.jpg "How epigenetics mechanism can effect health (adapted from https://commonfund.nih.gov/epigenomics/figure)")
 
-This exercise uses the dataset from the Cell publication by Hugo, Willy, et al., 2015 (DOI: 10.1016/j.cell.2015.07.061). The goal of this article was to identify differentially methylated regions and positions associated with treatment resistant melanomas.
-
-To this end, Illumina 450K Methylation array was performed in melanoma tumors from example patient pre and post MAPKi and BRAFi treatment with  different outcomes (sensitive and resistant). For each sample there is raw green and red colour arrays containing the summarized bead information generated bythe Illumina 450k scanner, i.e. there are 8 files in total:
+This exercise uses datasets from the Cell publication by Hugo, Willy, et al., 2015 (DOI: 10.1016/j.cell.2015.07.061). with the goal to identify differentially methylated regions and positions associated with treatment resistant melanomas.
```
Comment (bgruening) [inline topics/epigenetics/tutorials/ewas-suite/tutorial.md]: `. with` should this be a comma?

File: topics/epigenetics/tutorials/ewas-suite/tutorial.md [hunk 2]
```diff
@@ -26,9 +26,8 @@ The field of cancer genomics has demonstrated the power of massively parallel se
...
-
-To this end, Illumina 450K Methylation array was performed in melanoma tumors from example patient pre and post MAPKi and BRAFi treatment with  different outcomes (sensitive and resistant). For each sample there is raw green and red colour arrays containing the summarized bead information generated bythe Illumina 450k scanner, i.e. there are 8 files in total:
+This exercise uses datasets from the Cell publication by Hugo, Willy, et al., 2015 (DOI: 10.1016/j.cell.2015.07.061). with the goal to identify differentially methylated regions and positions associated with treatment resistant melanomas. 
+Datasets include Illumina 450K Methylation array performed in melanoma tumors in sample of patients pre and post MAPKi and BRAFi treatment with  different outcomes (sensitive and resistant). For each sample there is raw green and red colour arrays containing the summarized bead information generated bythe Illumina 450k scanner, i.e. there are 8 files in total:
```
Comment (bgruening) [inline topics/epigenetics/tutorials/ewas-suite/tutorial.md]: bythe -> by the

File: topics/epigenetics/tutorials/ewas-suite/tutorial.md [hunk 3]
```diff
@@ -130,22 +129,59 @@ Incomplete annotation of genetic variations such as single nucleotide polymorphi
...
+> 1. Run **minfi dropsnp** {% icon tool %}  on the output of minfi_maptogenome to remove the probes that contain either a SNP at the metylated loci interrogation or at the single nucleotide extension
 {: .hands_on}
 
 # DMPs and DMRs Identification
 
 The main goal of the EWAS suite is to simplify the way differentially methylated loci sites are detected. The EWAS suite contains **minfi dmp** {% icon tool %} tool detecting differentially methylated positions (DMPs) with respect to a phenotype covariate, and more complex **minfi dmr** {% icon tool %} solution for finding differentially methylated regions (DMRs). Genomic regions that are differentially methylated between two conditions can be tracked using a bumphunting algorithm. The algorithm first implements a t-statistic at each methylated loci location, with optional smoothing, then groups probes into clusters with a maximum location gap and a cutoff size to refer the lowest possible value of genomic profile hunted by our tool.
 
+In order the next we will need additional file that annotate illumina450K with genomic features, which we can obtain from UCSC.
```
Comment (bgruening) [inline topics/epigenetics/tutorials/ewas-suite/tutorial.md]: `In order the next` maybe just `In the next step we will need`

**Comment by bgruening** [comment]
That was supposed to be a comment. @kpoterlowicz I added a few comments inline.

**Comment by kpoterlowicz** [comment]
hi, thanks I will do proofreading now with my daughter (native speaker  :))  so might have another pull request

**Comment by shiltemann** [comment]
@kpoterlowicz thanks for the updates! 

I added some small formatting tweaks, and I think we probably don't need the screenshots? we try to avoid them as mush as possible, and you already have very good textual descriptions of the parameters, so I think that is good enough, what do you think?

**Comment by bgruening** [comment]
Thanks all!
ping @kpbioteam

**Pull request merged**